packages/ember/tests/helpers/helper_registration_test.js

```diff
@@ -1,112 +1,67 @@
+import { moduleFor, ApplicationTestCase } from 'internal-test-helpers';
 import { Controller, Service, inject } from 'ember-runtime';
-import { run } from 'ember-metal';
-import { compile } from 'ember-template-compiler';
-import {
-  Helper,
-  helper,
-  setTemplates,
-  setTemplate
-} from 'ember-glimmer';
-import { Application } from 'ember-application';
-import { Router } from 'ember-routing';
-import { jQuery } from 'ember-views';
-
-let App, appInstance;
-
-QUnit.module('Application Lifecycle - Helper Registration', {
-  teardown() {
-    run(() => {
-      if (App) {
-        App.destroy();
-      }
+import { Helper, helper } from 'ember-glimmer';
 
-      App = appInstance = null;
-      setTemplates({});
-    });
-  }
-});
+moduleFor('Application Lifecycle - Helper Registration', class extends ApplicationTestCase {
+  ['@test Unbound dashed helpers registered on the container can be late-invoked'](assert) {
```

Comment:
bound-ness for this test and the one following refers to the arguments passed to the helper. In this test, a raw string in the template is passed and in the second test a variable is passed.